app/models/email_digests/linkedin_profiles.rb

```diff
@@ -0,0 +1,15 @@
+class EmailDigests::LinkedinProfiles
+  def send_digest
+    last_month = Date.current.last_month
+    last_month = last_month.beginning_of_month..last_month.end_of_month
+    linkedin_profiles = Developers::ExternalProfile
+      .where(updated_at: last_month)
```

Comment:
I'm confused by this line. Don't we want to include anyone who _changed their job_ in the last month? `updated_at` will be when we updated the record.

Or are you thinking that we only update once per month so _any_ update will correspond to a job change? Let me know what you're thinking here!

Reply:
We are updating record only if there is any change in its LinkedIn profile data https://github.com/joemasilotti/railsdevs.com/blob/9aaf5da476c59619c00e72b1bfae96f8e4ae0487/app/models/developers/external_profiles/linkedin_profile_fetcher.rb#L26

That is the reason I am checking updated_at as it will be record will be updated only if developer's current company data is updated.

Reply:
Got it, that makes sense. So the first email will be _every_ developer with a LinkedIn profile and the next one will only send changes. Thanks!